### ISSUE TITLE
OmniCС shows strange CD

### ISSUE THREAD

**Issue by BigBont** [tullamods/OmniCC#360]
I do not know how to explain, I will show:

https://i.snipboard.io/4xSpPO.jpg

OmniCC shows 1d CD - Blizzard UI show 2d CD

**Comment by Goranaws** [comment]
I have encountered this issue before while building a Buff/Debuff addon . Most likely, OmniCC is rounding numbers up or down, differently than the Blizzard UI. 

Another possibility is that multi-day cooldowns are an edge case that might not be accounted for with current text formatting.

**Comment by Tuller** [comment]
I don’t think this one is the blizzard cooldown bug edge case, and is likely a case of OmniCC rounding a cooldown under 1.5 days old where the blizzard UI rounds up.
